packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -870,6 +870,30 @@ private static List<List<LatLng>> toHoles(List<List<Messages.PlatformLatLng>> da
     return holes;
   }
 
+  private static List<PatternItem> patternFromPigeon(
+      List<Messages.PlatformPatternItem> patternItems) {
+    if (patternItems.isEmpty()) {
+      return null;
+    }
+    final List<PatternItem> pattern = new ArrayList<>();
+    for (Messages.PlatformPatternItem patternItem : patternItems) {
+      switch (patternItem.getType()) {
+        case DOT:
+          pattern.add(new Dot());
+          break;
+        case DASH:
+          assert patternItem.getLength() != null;
+          pattern.add(new Dash(patternItem.getLength().floatValue()));
+          break;
+        case GAP:
+          assert patternItem.getLength() != null;
+          pattern.add(new Gap(patternItem.getLength().floatValue()));
+          break;
+      }
+    }
+    return pattern;
+  }
+
   private static List<PatternItem> toPattern(Object o) {
```

Comment:
This is dead code, and should be removed. Please always check for dead conversion code when doing Pigeon conversions.